src/main.rs

```diff
@@ -66,11 +65,9 @@ fn report_errors(results: Vec<file_operations::Result>) {
 }
 
 fn make_app_config<'a, 'b>() -> App<'a, 'b> {
-    let version = env!("CARGO_PKG_VERSION");
-
     let app = App::new("hermit")
-        .version(version)
-        .author("Bike Barn <https://github.com/bike-barn/hermit>")
+        .version(env!("CARGO_PKG_VERSION"))
```

Comment:
I'm not keen on unrelated changes sneaking into a topic-based PR, but I'll allow it.